python/lsst/afw/table/baseColumnView.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from ._baseColumnView import *
```

Comment:
I can, but I'd like to give @pschella a chance to defend the current procedure to see if he can convince Russel before we go to the trouble of creating an RFC. I'm fairly certain that not only afw but all of the upstream packages that he's wrapped use the same syntax.